packages/cli/src/init.ts

```diff
@@ -136,32 +137,25 @@ export async function initializeProject(
 export async function extendPackageJson(dirPath: string, projectName: string): Promise<void> {
     const packageJsonPath = path.join(dirPath, 'package.json');
 
-    const packageJsonObject: {
-        name?: string;
-        private?: boolean;
-        scripts?: { [key: string]: string };
-        devDependencies?: { [key: string]: string };
-    } = {};
-
-    packageJsonObject.name = projectName;
-    packageJsonObject.private = true;
-    packageJsonObject.scripts = {
-        lint: 'eslint ./**/*.ts*',
-        typecheck: 'tsc --noEmit',
-
-        // TODO: this is GitBook-specific
-        'publish-integrations-staging': 'gitbook publish .',
-    };
-
-    packageJsonObject.devDependencies = {
-        ...(packageJsonObject.devDependencies || {}),
-        [packageJSON.name]: `^${packageJSON.version}`,
-        [`@gitbook/runtime`]: 'latest',
-        '@cloudflare/workers-types': 'latest',
-        '@gitbook/tsconfig': 'latest',
+    const packageJsonObject: PackageJSON = {
+        name: projectName,
+        private: true,
+        scripts: {
+            lint: 'eslint --ext .js,.jsx,.ts,.tsx .',
+            typecheck: 'tsc --noEmit',
+            publish: 'gitbook publish .',
+        },
+        dependencies: {
+            '@gitbook/runtime': 'latest',
```

Comment:
I'm now doubting again if this should be a production dependency or not; can it remain a dev dep because `@gitbook/runtime` gets bundled into a JS file when the integration it published and deployed on our cloud infra? 🤔

Reply:
I'm confused, isn't that the main reason why it should be in `deps`? 😄  because it _should_ be bundled with the main code as it'll be essential for the proper execution. Same reason why `react` is usually in `deps` but something like `eslint` isn't

Reply:
Yeah, that's what I figured as well, hence the diff in this PR. But I was doubting it later because I noticed that the Formspree integration for instance depends on `@gitbook/runtime` but has it listed as a dev dependency ([src](https://github.com/GitbookIO/integrations/blob/8ace9af200dda345be3feab2b63a0a3277246243/integrations/formspree/package.json#L11)).

Reply:
I think it's probably because we don't mark `devDeps` as **external** during build process 🤔 

https://github.com/GitbookIO/integrations/blob/8ace9af200dda345be3feab2b63a0a3277246243/packages/cli/src/build.ts#L33